src/test/java/com/ericsson/ei/frontend/BackendInformationControllerTest.java

```diff
@@ -51,6 +51,8 @@
     private static final String BACKEND_INSTANCES_FILE_PATH = "src/test/resources/backendInstances/backendInstances.json";
     private static final String BACKEND_RESPONSE_INSTANCES_FILE_PATH = "src/test/resources/backendInstances/expectedResponseInstances.json";
 
+    private static final String FRONT_END_REST = "/backend";
```

Comment:
I'm confused with the naming of this variable, shouldn't it be BACK_END_REST?

Reply:
I agree, although it should be PATH_FOR_BACK_END_INFORMATION